docs/application/vstools-mac/overview.md

```diff
@@ -0,0 +1,150 @@
+# Visual Studio for Mac Extension for Tizen
+
+Visual Studio for Mac Extension for Tizen enables you to develop Tizen aplications easily using Visual Studio Code.
+
+## Prerequisites
+
+To work with Visual Studio for Mac Extension for Tizen, your computer must have:
+
+- At least 5.6 GB of available disk space
+- macOS Mojave 10.14: Community, Professional, and Enterprise.	
+- Visual Studio Mac 2019 (Community, Professional, Enterprise) Version 8.2 and higher.
+
+- Java Development Kit (JDK)
+
+  You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 10 to use 
+Tizen Baseline SDK. Make sure you download and install the EXACT version.
+
+  - [Oracle Java Development Kit(JDK) 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and [Java for OS X 2015-001](https://support.apple.com/kb/DL1572)
+ 
+  - OpenJDK 10 and OpenJFX: [OpenJDK 10 and OpenJFX Installation Guide](https://developer.tizen.org/development/articles/openjdk-10-and-openjfx-installation-guide#install-openjdk-for-macos).
+
+For Oracle JDK, Java for OS X 2015-001 is required to support legacy Java software on macOS.
+
+<a name="setup"></a>
+## Visual Studio for Mac Extension for Tizen Installation
+
+You must install the **Visual Studio for Mac Extension for Tizen** and the **Baseline SDK** to use the Tizen SDK tools.
```

Comment:
This section describes how to install the **Visual Studio for Mac Extension for Tizen** and the **Baseline SDK** to use the Tizen SDK tools.

Reply:
Done